android/src/main/java/nordpol/ui/NfcGuideView.java

```diff
@@ -44,7 +46,7 @@
  * nfcGuideView.setCurrentStatus(NfcGuideView.NfcGuideViewStatus.TRANSFERRING);
  */
 public class NfcGuideView extends RelativeLayout {
-    private static final int GUIDE_ITEMS_START_DISTANCE = 100;
+    private int guideItemsStartDistance = 50;
```

Comment:
Just setting a functional default, this is replaced with whatever is in the resources later